src/libs/KeyboardAvoidingView/index.android.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {KeyboardAvoidingView as KeyboardAvoidingViewComponent} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+
+function KeyboardAvoidingView({children}) {
```

Comment:
Please add docs to the top of each platform-specific file to explain the component and why it's platform-specific.